src/Microsoft.Health.Dicom.Core/Features/Common/ISecretStore.cs

```diff
@@ -3,16 +3,71 @@
 // Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
 // -------------------------------------------------------------------------------------------------
 
+using System;
+using System.Collections.Generic;
 using System.Threading;
 using System.Threading.Tasks;
 
 namespace Microsoft.Health.Dicom.Core.Features.Common;
 
+/// <summary>
+/// Represents a data store that securely stores named secrets.
+/// </summary>
 public interface ISecretStore
```

Comment:
Where `ISecretStore` will be used? I guess it's part of Export, and the code using it will be checked in later?

Reply:
The options for the blob store sink will be "sensitive," as it may containing a SasToken. So when validating the options, the sink provider will use the secret store to "classify" (ie. store) the sensitive info in the secret store.